samples/Samples.Elasticsearch.V5/Program.cs

```diff
@@ -362,6 +362,11 @@ private static object TaskResult(Task task)
             task.GetAwaiter().GetResult();
             var taskType = task.GetType();
 
+            while (!taskType.Name.StartsWith("Task"))
```

Comment:
What types show up here that we need this check? Just curious.

Reply:
On .NET Core, the async state machine can create the following type `System.Runtime.CompilerServices.AsyncTaskMethodBuilder<>+AsyncStateMachineBox<>`, which subclasses: `Task<>`.

Before this change, we would detect that it wasn't a `Task<>` type and we would return null, so instead of printing out the result we would print out a blank line.